### PR TITLE
Propagate starboard configs to Chromium on linux

### PR DESCRIPTION
**Pull request by andrewsavage1**
Fixes linux builds with targets using starboard

b/365546355

### CODE REVIEW

**Comment by andrewsavage1** [comment]
FYI @y4vor and @TyHolc this might help when building on linux using starboard headers in chromium targets

File: starboard/linux/x64x11/toolchain/BUILD.gn [hunk 1]
```diff
@@ -24,6 +24,7 @@ overridable_clang_toolchain("starboard") {
     clang_use_chrome_plugins = false
     enable_check_raw_ptr_fields = false
   }
+  propagates_configs = true
```
Comment (andrewsavage1) [inline starboard/linux/x64x11/toolchain/BUILD.gn:27]: Obviously only applies to this toolchain right now so other linux flavors will have issues

Comment (niranjanyardi) [reply starboard/linux/x64x11/toolchain/BUILD.gn:27]: Also , i think the files @y4vor is thinking of atleast in that ozone prototype were supposed to be built with chromium toolchain, this only applies to starboard toolchain. correct me if i'm wrong @y4vor

Comment (niranjanyardi) [reply starboard/linux/x64x11/toolchain/BUILD.gn:27]: talked offline, this makes sense. We are just propogating the public starboard configs to whatever targets depend on them . (Configs are private by default)

**Review by niranjanyardi — APPROVED** [review]

**Pull request merged**